crates/xilem_html/src/element/mod.rs

```diff
@@ -89,12 +87,13 @@ impl<El, ViewSeq> Element<El, ViewSeq> {
     ///
     /// If the name contains characters that are not valid in an attribute name,
     /// then the `View::build`/`View::rebuild` functions will panic for this view.
-    pub fn set_attr(
-        &mut self,
-        name: impl Into<Cow<'static, str>>,
-        value: impl Into<Cow<'static, str>>,
-    ) {
-        self.attributes.insert(name.into(), value.into());
+    pub fn set_attr(&mut self, name: impl Into<CowStr>, value: impl IntoAttributeValue) {
```

Comment:
Right, so given the new structure of the code, here an `unset_attr` or `remove_attr` could make sense. It doesn't seem terribly important though.

Reply:
The automatic removal is intentional when e.g. doing something like el.attr("checked", false) (or the switch from `true` to `false`), so that the `"checked"` attribute will be removed properly from the element.
Though explicit attribute removal makes sense as well I guess (to avoid something like `el.set_attr(name, false)` which doesn't immediately tell that it's just for removal of the attribute).

I've added that with the latest commit.

Thanks for the review!